apps/web/scripts/i18n-cleanup/main.ts

```diff
@@ -0,0 +1,34 @@
+import * as glob from "glob";
+import * as path from "path";
+
+import { cleanupTranslations } from "./cleanUpTranslations";
+import { findUsedKeys } from "./findUsedKeys";
+import { getAllTranslationKeys } from "./getAllTranslationKeys";
+
+export type TranslationValue = string | { one: string; other: string };
+export type TranslationObject = Record<string, TranslationValue>;
+export const KEYS_TO_NOT_REMOVE: readonly string[] = [] as const;
+
+const main = async (): Promise<void> => {
+  const sourceDir = path.resolve("../../../");
+  const translationsDir = path.resolve("public/static/locales");
+
+  console.info("Scanning source directory:", sourceDir);
+  console.info("Scanning translations directory:", translationsDir);
+
+  const sourceFiles = glob.sync(`${sourceDir}/**/*.{ts,tsx,js,jsx}`, {
```

Comment:
Might also want to include sql files here as we have some keys directly stored in db

Reply:
```suggestion
  const sourceFiles = glob.sync(`${sourceDir}/**/*.{ts,tsx,js,jsx,sql}`, {
```